packages/rfw/lib/src/flutter/material_widgets.dart

```diff
@@ -140,18 +140,18 @@ Map<String, LocalWidgetBuilder> get _materialWidgetsDefinitions => <String, Loca
     );
   },
 
-  // The ButtonBar widget in package:flutter/material.dart is planned to be deprecated
-  // in favor of the OverflowBar widget. This ButtonBar implementation uses the
-  // OverflowBar widget internally for backward compatibility. The ButtonBar
-  // widget in rfw package is not deprecated and will continue to be supported.
+  // The [ButtonBar] widget in Flutter's material library is planned to be deprecated
+  // in favor of the [OverflowBar] widget. This [ButtonBar] implementation uses the
+  // [OverflowBar] widget internally for backward compatibility. The [ButtonBar]
+  // widget in `rfw` package is not deprecated and will continue to be supported.
```

Comment:
Updated this section to match for markdown format issues:

https://github.com/flutter/packages/blob/5b48c446976fd5f61316d58ab3fcc9888423c2e6/packages/rfw/lib/src/flutter/material_widgets.dart#L57-L68